MagneticField/VolumeBasedEngine/interface/MagGeometry.h

```diff
@@ -75,5 +72,12 @@ class MagGeometry {
 
   bool cacheLastVolume;
   int geometryVersion;
+
+  // boundaries of internal barrel-endcap volume separation
+  float barrelRsq1;
+  float barrelRsq2;
+  float barrelZ0;
+  float barrelZ1;
+  float barrelZ2;
```

Comment:
(most) data members have prefix "the" in this class, which is good for distinguishing from local variables.
Please update for consistency.

Reply:
done